cirq/experiments/cross_entropy_benchmarking.py

```diff
@@ -107,44 +182,87 @@ def plot(self, ax: Optional[plt.Axes] = None,
     def depolarizing_model(self) -> CrossEntropyDepolarizingModel:
         """Fit a depolarizing error model for a cycle.
 
-        Fits an exponential model f = S * p^d, where d is the number of cycles
+        Fits an exponential model f = S * p**d, where d is the number of cycles
         and f is the cross entropy fidelity for that number of cycles,
         using nonlinear least squares.
 
         Returns:
-            A DepolarizingModel object, which has attributes `coefficient`
-            representing the value S, `decay_constant` representing the value
-            p, and `covariance` representing the covariance in the estimation
-            of S and p in that order.
+            A CrossEntropyDepolarizingModel object, which has attributes
+            `coefficient` representing the value S, `decay_constant`
```

Comment:
The original text you modified appears outdated. The attributes are "spam_depolarization" and "cycle_depolarization", not "coefficient" and "decay_constant". Please update.

(Applies to Returns: section in the docstring for `purity_depolarizing_model` as well.)

Reply:
Done.